mlflow/pyspark/ml/__init__.py

```diff
@@ -256,6 +269,20 @@ def _get_param_map(instance):
     }
 
 
+def _get_stage_type_params(instance):
+    """
+    Returns a map of param name and param value where
+    the param value is of type pyspark.ml.param.Params
```

Comment:
Get the param map of the instance where param value is type  `pyspark.ml.param.Params`

Reply:
Done.